src/router.rs

```diff
@@ -1,8 +1,9 @@
-use actix_web::web::{post, resource, ServiceConfig};
```

Comment:
This module (`router`) would be a nice candidate for the `server` module suggested above